manuscript/algorithm.tex

```diff
@@ -160,9 +160,18 @@ \subsubsection{\gls{MARKAL}}
 The variables and the functions are listed in detail in the reference
 \cite{fishbone_markal_1981} \cite{abilock_users_1979}. 
 
+
 \paragraph{Multi-objective linear-programming.}
 The optimization algorithm in \gls{MARKAL} is a collection
 of objective functions, subject to a collection of constraints.
+Multi-objective linear programming is used in various applications,
+such as economics, finance, engineering design, and power systems
+\cite{dorfman_linear_1958}.
```

Comment:
you must update the bib file. "
  Citation `bergamaschi_a_2011' on page 4 undefined on input line 227
  Citation `dorfman_linear_1958' on page 4 undefined on input line 169
  Citation `engel_can_1994' on page 4 undefined on input line 226
  Citation `krolzig_a_2002' on page 4 undefined on input line 227
  Citation `lauren_stochastic_2015' on page 4 undefined on input line 197
  Citation `melo_gaussian_2012' on page 5 undefined on input line 248"